app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -654,6 +674,26 @@ class BrowserFragment :
         }
     }
 
+    private fun openSelectBrowser() {
```

Comment:
I noticed that there is a regression with this feature where we are not able to find other others to "open with".

I looked into it when reviewing this patch, but I can't figure out what's the cause since the code here hasn't changed.